src/components/map/map.js

```diff
@@ -41,6 +37,10 @@ function Map({ incidents, onCityClick }) {
 
   const onLoad = React.useCallback(function callback(mapInstance) {
     setMap(mapInstance);
+    mapInstance.setCenter({
```

Comment:
Since we're setting the center here, can we also move the `zoom` property up, for consistency?